performance/buildSrc/src/main/kotlin/RunKotlinNativeTask.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the license/LICENSE.txt file.
+ */
+
+import groovy.lang.Closure
+import org.gradle.api.DefaultTask
+import org.gradle.api.Task
+import org.gradle.api.tasks.TaskAction
+import org.jetbrains.kotlin.gradle.plugin.KotlinTarget
+import javax.inject.Inject
+import java.io.File
+
+open class RunKotlinNativeTask @Inject constructor(
+        private val curTarget: KotlinTarget
+): DefaultTask() {
+
+    var buildType = "RELEASE"
+    var workingDir: Any = project.projectDir
+    var outputFileName: String? = null
+    private var curArgs: List<String> = emptyList()
+    private val curEnvironment: MutableMap<String, Any> = mutableMapOf()
+
+    fun args(vararg args: Any) {
+        curArgs = args.map { it.toString() }
+    }
+
+    fun environment(map: Map<String, Any>) {
```

Comment:
`Map<String, String>`?

Reply:
As I understood this type is predefined in plugin. I need such type because project.exec.environment has such type.